README.md

```diff
@@ -5,13 +5,19 @@ This is an application that is used to handle simple mathematical operations
 ## Languages
 
 - React
+- Html
+- Css
 
 ## Josphat Kiploman
 
 - [@Github](https://github.com/Josphat205)
 
 - [@Linkedin](https://www.linkedin.com/in/josphat-kiploman-797430236/)
 
+## 🤝 Dependencies
+- babel
+- webpack
+- Big js
```

Comment:
- [ ] Please add more details to your readme file like how to clone the repo and install the packages and run the app, also please add the project description like what is the purpose of doing it.